### ISSUE TITLE
training data

### ISSUE THREAD

**Issue by eeric** [megvii-model/MOTR#9]
inputing 5 frames to train, they were the requirement continuous, weren't they？

**Comment by dbofseuofhust** [comment]
yes, inputs should be  continuous, but in our implemention, there may sample frames from different videos, we just ignore the influense, because for frame from other video, all objects are "new appear" objects and will help leaning process of "empty queries".

**Comment by dbofseuofhust** [comment]
> inputing 5 frames to train, they were the requirement continuous, weren't they？

yes, inputs should be continuous, but in our implemention, there may sample frames from different videos, we just ignore the influence, because for frame from other video, all objects are "new appear" objects and will help the leaning process of "empty queries".

**Comment by eeric** [comment]
ok, great solution!
